apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx

```diff
@@ -1,31 +1,137 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import PropTypes from 'prop-types';
 import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
 import i18n from '@cdo/locale';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
 import StylizedBaseDialog from '@cdo/apps/componentLibrary/StylizedBaseDialog';
-import CodeReviewGroupsLoader from '@cdo/apps/templates/codeReviewGroups/CodeReviewGroupsLoader';
 import CodeReviewGroupsStatusToggle from '../codeReviewGroups/CodeReviewGroupsStatusToggle';
+import CodeReviewGroupsManager from '@cdo/apps/templates/codeReviewGroups/CodeReviewGroupsManager';
 
 const DIALOG_WIDTH = 1000;
+const STATUS_MESSAGE_TIME_MS = 5000;
+
+const SUBMIT_STATES = {
+  DEFAULT: 'default',
+  SUBMITTING: 'submitting',
+  SUCCESS: 'success',
+  ERROR: 'error'
+};
+
+const LOADING_STATES = {
+  LOADING: 'loading',
+  LOADED: 'loaded',
+  ERROR: 'error'
+};
 
 export default function ManageCodeReviewGroups({
```

Comment:
Perhaps CodeReviewGroupsDialog would be a clearer name?

Reply:
I was matching the naming convention used in `ManageStudents`, which is a similar button/dialog component. There are a lot of components in that directory starting with Manage that don't follow this pattern though, so I think that sounds good!